### DISCUSSION TITLE
Missing ToBitmapSource

### DISCUSSION THREAD

**Discussion by mpoelzl** [dlemstra/Magick.NET#671] [General] [answered]
Hello!

I have just upgraded Magick.NET-Q8-AnyCPU.dll for .NET 4 from 7.16.0.0 to the latest version but now I am missing the MagickImage.ToBitmapSource method. What am I doing wrong? THis is for .NET 4.5.2

Thanks,

Mike

**Comment by dlemstra** [comment, marked as answer]
With a search you would have found #666, #657 or #652.

**Comment by mpoelzl** [comment]
Hello!

Thanks for the reply this got me the solution. I did try the search but searching for ToBitmapSource didn't show me the above issues.

In any case your work is much appreciated!

Mike

**Reply by dlemstra** [reply]
Happy to hear that you got this resolved. When I search inside discussions for `ToBitmapSource` I find one of these issues. Strange that you did not find any issue.

**Comment by mpoelzl** [comment]
Ah yes, I was searching in issues and not discussions.

Kind Regards!
